optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -411,6 +423,20 @@ const plotHistory = (
       type: "scatter",
     })
   }
+  plotData.push({
+    x: infeasibleTrials.map(getAxisX),
+    y: infeasibleTrials.map(
+      (t: Trial): number => target.getTargetValue(t) as number
+    ),
+    name: "Infeasible Trial",
+    marker: {
+      size: markerSize,
+      color: "#cccccc",
```

Comment:
[nit] `#cccccc` might be a bit too light in Dark mode. What do you think about using `#666666` in dark mode instead?

<img width="1757" alt="Screenshot 2023-07-28 10 00 04" src="https://github.com/optuna/optuna-dashboard/assets/5564044/c0a07b08-b117-4824-8643-2102d594e174">
<img width="1744" alt="Screenshot 2023-07-28 9 59 58" src="https://github.com/optuna/optuna-dashboard/assets/5564044/daf42085-0ebd-4ac8-9282-e95bada302f4">

```suggestion
      color: mode === "dark" ? "#666666" : "#cccccc",
```

Then the history plot will be rendered like below:

<img width="1756" alt="Screenshot 2023-07-28 10 07 13" src="https://github.com/optuna/optuna-dashboard/assets/5564044/e8ec5eb1-2d9b-4e32-94b6-2e72ea589ec3">